test/src/main/java/org/apache/accumulo/test/start/KeywordStartIT.java

```diff
@@ -109,34 +109,35 @@ public void testCheckDuplicates() {
   public void testExpectedClasses() {
     assumeTrue(new File(System.getProperty("user.dir") + "/src").exists());
     TreeMap<String,Class<? extends KeywordExecutable>> expectSet = new TreeMap<>();
-    expectSet.put("admin", Admin.class);
-    expectSet.put("check-compaction-config", CheckCompactionConfig.class);
-    expectSet.put("check-server-config", CheckServerConfig.class);
-    expectSet.put("compaction-coordinator", CoordinatorExecutable.class);
-    expectSet.put("compactor", CompactorExecutable.class);
-    expectSet.put("config-upgrade", ConfigPropertyUpgrader.class);
-    expectSet.put("convert-config", ConvertConfig.class);
-    expectSet.put("create-token", CreateToken.class);
-    expectSet.put("ec-admin", ECAdmin.class);
-    expectSet.put("gc", GCExecutable.class);
-    expectSet.put("generate-splits", GenerateSplits.class);
-    expectSet.put("help", Help.class);
-    expectSet.put("info", Info.class);
-    expectSet.put("init", Initialize.class);
-    expectSet.put("login-info", LoginProperties.class);
-    expectSet.put("manager", ManagerExecutable.class);
-    expectSet.put("master", org.apache.accumulo.manager.MasterExecutable.class);
-    expectSet.put("minicluster", MiniClusterExecutable.class);
-    expectSet.put("monitor", MonitorExecutable.class);
-    expectSet.put("rfile-info", PrintInfo.class);
-    expectSet.put("wal-info", LogReader.class);
-    expectSet.put("shell", Shell.class);
-    expectSet.put("tserver", TServerExecutable.class);
-    expectSet.put("version", Version.class);
-    expectSet.put("zookeeper", ZooKeeperMain.class);
-    expectSet.put("create-empty", CreateEmpty.class);
-    expectSet.put("split-large", SplitLarge.class);
-    expectSet.put("zoo-zap", ZooZap.class);
+    expectSet.put(Admin.EXE_NAME, Admin.class);
+    expectSet.put(CheckCompactionConfig.EXE_NAME, CheckCompactionConfig.class);
```

Comment:
The whole point of the checks here is to ensure that the string isn't changed. By doing this level of indirection, if the value of the constant `EXE_NAME` is changed, then this test will pass, even if it shouldn't have been changed. This test no longer guards against such changes.
This is now effectively just checking that `SomeClass.EXE_NAME == SomeClass.keyword()` for every `SomeClass`. But, that's basically tautological the way all the classes have changed.

Reply:
I'll close this and submit a PR that updates the test documentation to include this. I was unclear on the reason for the test being constructed as it is. When adding / changing a command it is not obvious that the KeywordIT test must also be changed - well, at least until if fails, which is its purpose, but frustrating from a development perspective.